plugin.yaml

```
@@ -1,8 +1,9 @@
 name: "ksec"
+version: "0.1.3"
 usage: "Manage Kubernetes Secrets through Helm"
 description: "Helm plugin that simplifies the management of Kubernetes Secrets"
 ignoreFlags: false
 useTunnel: false
 command: "$HELM_PLUGIN_DIR/ksec"
 hooks:
-  install: "$HELM_PLUGIN_DIR/scripts/install-binary.sh"
+  install: "cd $HELM_PLUGIN_DIR; scripts/install-binary.sh"
```

nit: I'd prefer this to use &&. But don't know if that will cause incompatibilities

```suggestion
  install: "cd $HELM_PLUGIN_DIR && scripts/install-binary.sh"
```